go/vt/vtctl/reparentutil/planned_reparenter.go

```diff
@@ -103,18 +119,23 @@ func (pr *PlannedReparenter) ReparentShard(ctx context.Context, keyspace string,
 	if opts.NewPrimaryAlias == nil && opts.AvoidPrimaryAlias == nil {
 		shardInfo, err := pr.ts.GetShard(ctx, keyspace, shard)
 		if err != nil {
+			prsFailureCounter.Add(statsLabels, 1)
 			return nil, err
 		}
 
 		opts.AvoidPrimaryAlias = shardInfo.PrimaryAlias
 	}
 
+	startTime := time.Now()
 	ev := &events.Reparent{}
 	defer func() {
+		reparentShardOpTimings.Add("PlannedReparentShard", time.Since(startTime))
```

Comment:
Repeating comment from ERS: should we track the timings of all operations or success only? 🤔

Reply:
Same comment as for ERS. Let's do all.